_data/sharanp98/sharanp98.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sharan P # No longer than 18 characters
+institution: Sree Chitra Thirunal College of Engineering 🚩 # no longer than 58 characters
+profile_pic: sharan.jpeg # Name and extension of your profile picture(ex. mona.png)
+quote: "From barely knowing how to write code to paid internships, college helped me define who I am!" # no longer than 100 characters

```

Comment:
please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format